client/src/components/StarsRating/__tests__/__snapshots__/index.spec.js.snap

```diff
@@ -5,19 +5,46 @@ exports[`<StarsRating /> should render component 1`] = `
   className="stars-rating-container"
 >
   <label
-    className="rating-label"
+    className="rating-label "
```

Comment:
Maybe is better to implement the error class that it will not affect the snapshots without error? I do not like this extra space in the end...